bitcoin/src/blockdata/witness.rs

```diff
@@ -269,17 +267,22 @@ impl Witness {
         self.content[end_varint..end_varint + new_element.len()].copy_from_slice(new_element);
     }
 
+    /// Pushes a [`crate::ecdsa::Signature`] as a new element on the witness (the DER-encoded
+    /// secp256k1::ecdsa::Signature along with the hash type), requires an allocation.
```

Comment:
Is it really needed to repeat that the witness may reallocate (it's not even required)? At first I  thought there's one more allocation beside that but there isn't.

Reply:
Ah I just copied the docs from the other function, I wasn't sure why it was even mentioned on them. Will remove.

Reply:
Oh, right, the other function has the same problem.